packages/config-utils/proxy.js

```diff
@@ -26,15 +26,29 @@ module.exports = ({
     appUrl = [],
     publicPath,
     proxyVerbose,
-    useCloud = false
+    useCloud = false,
+    useStage = false,
+    target = ''
```

Comment:
I guess we should add these to the documentation (readme).

And config does not pass all attributes to proxy automatically so add it there too https://github.com/RedHatInsights/frontend-components/blob/8ffea0df0c366922c180a4f3a210d3e472a91e56/packages/config/src/config.js#L180

and shouldn't be stage indicated just by `env=` `stage-stable` or `stage-beta`? 🤔

Reply:
I'll add to the docs and make sure the props are passed.

The `env` is used to determine which build repo branches to checkout. `stage-stable` and `stage-beta` don't exist, but I suppose I could use them and then replace `stage` with `qa`.

Reply:
That would be awesome! 🍹

Reply:
But I think stage branches can exist, if repos configure them.. @Hyperkid123 ? 🤔 

(Not for Chrome though)

Reply:
It does not really matter from which branch you run the proxy. And chrome does not have a stage branch in the build repo. Stage env usually use QA builds. For chrome and every other application.

Reply:
Yeah, but I think if application create an stage branch, it would be served from that, wouldn't it?